daml-lf/spec/contract-id.rst

```diff
@@ -64,37 +65,29 @@ where
   uniqueness of the contract ID in a distributed ledger.
 
 
-Discriminator freshness
------------------------
+Local/Global Contract IDs
+-------------------------
 
 In a transaction we distinguish two kinds of contract IDs:
 
 * The *local* contract IDs are the IDs of the contracts created by the
   transaction.
 
 * The *global* contract IDs are the contract IDs that:
-   
+
    * appear in the commands that produced the transaction. This
      includes the IDs of the exercised contract, together with all the
      IDs referenced in the arguments of the create and exercise
      commands;
    * that are fetched or looked up by key unless they are local;
-   * are referenced in the input contracts.
+   * are referenced in the payloads of the fetched contracts.
```

Comment:
nit: arguably input is more correct since an exercise does not have a direct fetch

Reply:
```suggestion
   * are referenced in the input contracts.
```